.github/actions/spelling/allow.txt

```diff
@@ -1,559 +1,561 @@
-mrm
-msbuild
-msdata
-MSDN
-msi
-msiexec
-msix
-msixbundle
-msixinfo
-msixsdk
-msixsdkx
-msixtest
-msrc
-Multifile
-Multimatch
-mutex
-mutexes
-namespace
-namespaces
-Nelon
-netcoreapp
-newid
-NOCLOSEPROCESS
-nodiscard
-noexcept
-nomem
-NOMINMAX
-NONAME
-nonexistentsetting
-NONINFRINGEMENT
-NOTHROW
-NOTIMPL
-NOTNULL
-npos
-NTFS
-NTSTATUS
-nuget
-nullptr
-nullsoft
-nupkg
-nuspec
-nunit
-OAuth
-ofstream
-opencode
-opensource
-ostream
-ostringstream
-OSVERSIONINFOEXW
-outfile
-OUTOFMEMORY
-OWC
-PACKAGESSCHEMA
-Params
-params
-parentidx
-pathpart
-Pathto
-PBYTE
-pch
-PCWSTR
-pdb
-PEVENT
-pfp
-PGP
-PII
-pipssource
-placeholders
-png
-posix
-powershell
-PPV
-pplx
-pragma
-PRECONFIG
-preindexed
-prepareforpackaging
-PRIMARYKEY
-prioritization
-PRODUCTNAME
-PRODUCTVERSION
-psz
-ptr
-publiccontainer
-PUCHAR
-PVOID
-pwa
-QCol
-RAII
-rclsid
-rdbuf
-README
-READONLY
-READWRITE
-realloc
-REALTIME
-Redistributable
-REFCLSID
-regex
-regexp
-removemanifest
-repolibtest
-rescap
-resheader
-resmimetype
-RESTSOURCE
-resw
-resx
-roadmap
-robuffer
-rowcount
-rowid
-rubengustorage
-ruleset
-runsettings
-runtimes
-safecast
-SANITIZERS
-savepoint
-screenshots
-SCROLLER
-SCROLLVIEWER
-sdk
-seekg
-seinfo
-selectany
-SERVICEPACKMAJOR
-SERVICEPACKMINOR
-setfill
-setschemaversion
-setvariable
-setw
-shcore
-shellapi
-SHELLEXEC
-SHELLEXECUTEINFO
-SHELLEXECUTEINFOA
-SHELLEXECUTEINFOW
-Shlwapi
-signtool
-silentwithprogress
-simplesave
-simpletest
-sizeof
-sln
-SNAME
-snprintf
-sourced
-Specv
-sprintf
-sql
-sqlbuilder
-sqlite
-sqliteicu
-sqliteindex
-sqliteindexsource
-sqlitewrapper
-srand
-src
-srwlock
-sscanf
-sstream
-STATFLAG
-STATSTG
-stdcall
-stderr
-stdexcept
-stdout
-STGM
-stmt
-stoi
-stoll
-stoull
-strcoll
-streamoff
-streampos
-stricmp
-strikethrough
-STRINGID
-STRINGIFY
-STRINGIZE
-stringstream
-strstr
-subcontext
-SUBLANG
-subresource
-subselect
-substr
-SWIPECONTROL
-SYMED
-Sys
-sz
-TARG
-TARGETDIR
-targetentrypoint
-targetnametoken
-tdbuild
-tdd
-tellg
-tempdb
-terabyte
-testcontainer
-testmoniker
-Testrun
-testsettingname
-TEXTFORMAT
-TEXTINCLUDE
-there're
-Timeline
-todo
-tolower
-toupper
-towlower
-TRACELOGGING
-triaged
-trunc
-TRUSTEDPEOPLE
-tt
-ttl
-typedef
-typename
-UAC
-uap
-UBool
-UBreak
-ubrk
-uchar
-ucol
-UCollation
-UCollator
-UError
-UIA
-UIF
-uint
-Uknown
-ulong
-ULONGLONG
-uncomment
-und
-undef
-unicode
-UNICODESTRING
-uninstall
-uninstalling
-Unregister
-updatemanifest
-UPLEVEL
-upvote
-uregex
-URegular
-uri
-url
-urlmon
-userguide
-USERPROFILE
-usersources
-utext
-utf
-uuidof
-validator
-valijson
-vcvars
-vcxitems
-vcxproj
-vdproj
-versioned
-VERSIONINFO
-versioning
-VERYSILENT
-VFT
-visualstudio
-vm
-VOS
-vso
-wapproj
-wchar
-wcout
-wcsicmp
-webpage
-wekyb
-wil
-WINAPI
-WINEVENT
-winget
-wininet
-winmeta
-winres
-winrt
-winsqlite
-wix
-wmain
-woah
-wofstream
-workaround
-workflow
-wostringstream
-wostream
-wpfn
-wrl
-WStr
-wstring
-wstringstream
-www
-xamarin
-xlang
-xml
-xmlns
-XElement
-XName
-XResource
-xsd
-XTOKEN
-yaml
-yml
-yy
-zh
+abi

```

Comment:
I wonder if this file having every line changed is what has lead to so many spelling failures.  I suspect that github has merged this file and mucked about with the line endings.

Reply:
There's a 0.0.18. I need to send a PR.

I thought that I had fixed it to complain but tolerate this case:
```
winget-cli/.github/actions/spelling/allow.txt:    ASCII text, with CRLF line terminators
winget-cli/.github/actions/spelling/excludes.txt: ASCII text
winget-cli/.github/actions/spelling/expect.txt:   ASCII text
winget-cli/.github/actions/spelling/patterns.txt: ASCII text
winget-cli/.github/actions/spelling/reject.txt:   ASCII text
```

Anyway, dos2unix on the file does fix this. https://github.com/jsoref/winget-cli/actions/runs/962433068

Reply:
For reference, c0f1081ae5def579847148c3bee7926c1bc1d6c6 changed the line endings (so it wasn't a merge).

Reply:
Indeed, in 0.0.18 the tool would have tolerated the changed line endings:https://github.com/check-spelling/winget-cli/runs/2890223690.
> Warning: .github/actions/spelling/allow.txt: line 1, columns 3-4, Warning - entry has unexpected whitespace (whitespace-in-dictionary)
...
(Yes, it's very noisy...)

That said, I'm not sure you want the line ending changes commits...